src/gluonts/nursery/temporal_hierarchical_forecasting/model/cop_deepar/_estimator.py

```diff
@@ -208,6 +211,17 @@ def __init__(
 
         print(f"Distribution output: {base_estimator_hps['distr_output']}")
 
+        if "impute_missing_values" not in base_estimator_hps:
+            base_estimator_hps["impute_missing_values"] = impute_missing_values
+
+        if "imputation_method" not in base_estimator_hps:
+            base_estimator_hps["imputation_method"] = imputation_method
```

Comment:
One problem here is that the given imputation method will be applied to base estimators at all temporal levels. For eg. if you use `RollingMeanValueImputation`, each temporal level might need a different imputation window size that is suitable for its granularity.

Reply:
Thank you for this observation. I think we can handle this in basically the same way as we were handling the lags upper bound via `temporal_hierarchy.agg_multiples`.